frontend/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

```diff
@@ -41,146 +49,46 @@ export function applyStreamlitTheme(config: any, theme: Theme): any {
     axis: {
       labelFontSize: theme.fontSizes.twoSmPx,
       labelFontWeight: theme.fontWeights.normal,
-      labelColor: theme.colors.fadedText60,
+      labelColor: getGray70(theme),
       labelFontStyle: "normal",
       titleFontWeight: theme.fontWeights.normal,
       titleFontSize: theme.fontSizes.smPx,
-      titleColor: theme.colors.fadedText60,
+      titleColor: getGray70(theme),
       titleFontStyle: "normal",
       ticks: false,
-      gridColor: theme.colors.fadedText05,
+      gridColor: getGray30(theme),
       domain: false,
       domainWidth: 1,
-      domainColor: theme.colors.fadedText05,
+      domainColor: getGray30(theme),
       labelFlush: true,
       labelFlushOffset: 1,
       labelBound: false,
       labelLimit: 100,
-      titlePadding: 16,
-      labelPadding: 16,
-      labelSeparation: 4,
+      titlePadding: theme.spacing.lgPx,
+      labelPadding: theme.spacing.lgPx,
+      labelSeparation: theme.spacing.twoXSPx,
       labelOverlap: true,
     },
     legend: {
       labelFontSize: theme.fontSizes.smPx,
       labelFontWeight: theme.fontWeights.normal,
-      labelColor: theme.colors.bodyText,
+      labelColor: getGray70(theme),
       titleFontSize: theme.fontSizes.smPx,
       titleFontWeight: theme.fontWeights.normal,
       titleFontStyle: "normal",
-      titleColor: theme.colors.fadedText60,
-      titlePadding: 12,
-      labelPadding: 16,
-      columnPadding: 8,
-      rowPadding: 4,
+      titleColor: getGray70(theme),
+      titlePadding: theme.spacing.mdPx,
+      labelPadding: theme.spacing.lgPx,
+      columnPadding: theme.spacing.smPx,
+      rowPadding: theme.spacing.twoXSPx,
       padding: 7,
       symbolStrokeWidth: 4,
```

Comment:
nit: we probably can use `twoXSPx` here or?

Reply:
I consciously didn't change symbolStrokeWidth because it didn't seem like a spacing primitive. I can change this if you disagree, just react to this comment and I'll change it or comment again.

Reply:
ahh, I see. Yes that sounds reasonable 👍